express/src/validators/emailOtpValidator.ts

```diff
@@ -1,34 +1,23 @@
 import {NextFunction, Request, Response} from "express";
```

Comment:
This is middleware, so should live in `src/middleware`

Reply:
Ha! scrub the comment I just wrote above 😆